ref(angular): Update Angular SDK docs for `@sentry/angular-ivy` (#6356)
getsentry/sentry-docs: src/platform-includes/getting-started-verify/javascript.angular.mdx

Trigger a test error somewhere in your Angular app, for example in your main app component:

```html {filename: app.component.html}
<button (click)="throwTestError()">Test Sentry Error</button>
```

Then, in your `app.component.ts` add:

```javascript {filename: app.component.ts}
public throwTestError(): void {
  throw new Error("Sentry Test Error");
}
```

<Note>

> Errors triggered from within Browser DevTools are sandboxed, so will not trigger an error handler. Place the snippet directly in your code instead.
> 

Reviewer: ```suggestion
Errors triggered from within Browser DevTools are sandboxed and won't trigger an error handler. Place the snippet directly in your code instead.

```
